csrankings-o.csv

```diff
@@ -28,6 +28,7 @@ Ohad Shamir,Weizmann Institute of Science,https://www.weizmann.ac.il/pages/searc
 Oisin Mac Aodha,University of Edinburgh,https://homepages.inf.ed.ac.uk/omacaod,IfZBjkUAAAAJ
 Ojaswa Sharma,IIIT Delhi,https://www.iiitd.ac.in/ojaswa,jikZgTsAAAAJ
 Okan Yurduseven,Queen's University Belfast,https://www.qub.ac.uk/schools/eeecs/Connect/Staff/BusinessCard/?name=o.yurduseven,sjHXrh8AAAAJ
+Okko Johannes Räsänen,Tampere University,https://www.tuni.fi/en/okko-rasanen,iMVBRk0AAAAJ

```

Comment:
Research fellow. Please remove this and similar instances.

Reply:
Similarly https://researchportal.tuni.fi/en/persons/okko-r%C3%A4s%C3%A4nen will show Okko as both an Academy Research Fellow and Tenure Track Assistant Professor.